src/SFA.DAS.AssessorService.Data/OrgansiationStandardRepository.cs

```diff
@@ -0,0 +1,41 @@
+﻿using Dapper;
+using SFA.DAS.AssessorService.Application.Interfaces;
+using SFA.DAS.AssessorService.Domain.Entities;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.AssessorService.Data
+{
+    public class OrgansiationStandardRepository : Repository, IOrgansiationStandardRepository
```

Comment:
Typo on the repository and interface - OrganSIAtion

Reply:
Fixed